### PR TITLE
sort folder styles

### PR DESCRIPTION
**Pull request by Hugo-Hontans**

### CODE REVIEW

File: styles/variables.scss [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-$blockframes-theme-primary: mat-palette($mat-bf-blue);
-$blockframes-theme-accent: mat-palette($mat-bf-lightblue);
-$blockframes-theme-warn: mat-palette($mat-bf-crimson);
-
 $primary: mat-color($blockframes-theme-primary);
 $accent: mat-color($blockframes-theme-accent);
 $warn: mat-color($blockframes-theme-warn);
+
+// CSS color variables
+:root {
```
Comment (GrandSchtroumpf) [inline styles/variables.scss]: This should be remove at some point no ?

Comment (Hugo-Hontans) [reply styles/variables.scss]: We can use directly variables like $primary in css but it requires to import variables.scss in each css. And like this we can add others colors

Comment (Hugo-Hontans) [reply styles/variables.scss]: And change color easily for a dark-theme

Comment (GrandSchtroumpf) [reply styles/variables.scss]: I think it would be better to use scss variables if possible, because most of documentation will do that.

**Pull request merged**